python/ptest/cases/gomspace_checkout_case.py

```diff
@@ -0,0 +1,104 @@
+
+        curout = [
+            simulation.flight_controller.read_state("gomspace.curout.output1"),
+            simulation.flight_controller.read_state("gomspace.curout.output2"),
+            simulation.flight_controller.read_state("gomspace.curout.output3"),
+            simulation.flight_controller.read_state("gomspace.curout.output4"),
+            simulation.flight_controller.read_state("gomspace.curout.output5"),
+            simulation.flight_controller.read_state("gomspace.curout.output6")]
+
+        output = [
+            simulation.flight_controller.read_state("gomspace.output.output1"),
+            simulation.flight_controller.read_state("gomspace.output.output2"),
+            simulation.flight_controller.read_state("gomspace.output.output3"),
+            simulation.flight_controller.read_state("gomspace.output.output4"),
+            simulation.flight_controller.read_state("gomspace.output.output5"),
+            simulation.flight_controller.read_state("gomspace.output.output6")]
+
+        for n in range(0, len(output)):
+            out_n = output[n]
+            if out_n < 0.5 or out_n > 3.0:
+                print("Output-" + n +
+                      " is out of range [0.3, 0.5] at: " + out_n)
+
+        wdt_i2c_time_left = simulation.flight_controller.read_state(
+            "gomspace.wdt_i2c_time_left")
+        counter_wdt_i2c = simulation.flight_controller.read_state(
+            "gomspace.counter_wdt_i2c")
+        counter_boot = simulation.flight_controller.read_state(
+            "gomspace.counter_boot")
+
+        temp = [
+            simulation.flight_controller.read_state("gomspace.temp.output1"),
+            simulation.flight_controller.read_state("gomspace.temp.output2"),
+            simulation.flight_controller.read_state("gomspace.temp.output3"),
+            simulation.flight_controller.read_state("gomspace.temp.output4")]
+
+        bootcause = simulation.flight_controller.read_state(
+            "gomspace.bootcause")
+        battmode = simulation.flight_controller.read_state("gomspace.battmode")
+        pptmode = simulation.flight_controller.read_state("gomspace.pptmode")
+
+        # writable fields
+
+        simulation.flight_controller.read_state(
+            "gomspace.power_cycle_output1_cmd")
+        simulation.flight_controller.read_state(
+            "gomspace.power_cycle_output2_cmd")
+        simulation.flight_controller.read_state(
+            "gomspace.power_cycle_output3_cmd")
+        simulation.flight_controller.read_state(
+            "gomspace.power_cycle_output4_cmd")
+        simulation.flight_controller.read_state(
+            "gomspace.power_cycle_output5_cmd")
+        simulation.flight_controller.read_state(
+            "gomspace.power_cycle_output6_cmd")
+
+        simulation.flight_controller.read_state("gomspace.pv1_cmd")
+        simulation.flight_controller.read_state("gomspace.pv2_cmd")
+        simulation.flight_controller.read_state("gomspace.pv3_cmd")
```

Comment:
We don't really expect to use these particular commands in orbit so we might as well just not test these.